mubench.pipeline/tasks/implementations/compile.py

```diff
@@ -79,7 +79,7 @@ def process_project_version(self, project: Project, version: ProjectVersion):
                 logger.info("Compiling project...")
                 logger.debug("Copying patterns to source directory...")
                 self.__copy(version.patterns, sources_path)
-                self._compile(version.compile_commands, build_path, project_compile.dependencies_path)
+                self._compile(version.compile_commands, build_path, project_compile.dependencies_path, self.checkouts_base_path)
```

Comment:
Please use the `compiles_base_path` here. Although, currently, both are identical, we're handling files in the compiles directory here.